drivers/bbram/Kconfig.it8xxx2

```diff
@@ -8,3 +8,11 @@ config BBRAM_IT8XXX2
 	help
 	  This option enables the BBRAM driver for RISCV_ITE family of
 	  processors.
+
+config BBRAM_IT8XXX2_EMUL
```

Comment:
Non-blocking, but I would prefer each chipset changes to be split into separate commit.

Reply:
Done